docs/traceability/index.md

```diff
@@ -59,7 +59,8 @@ Below is the list of Central configuration properties in YAML and their correspo
 | central.proxyURL                       | CENTRAL_PROXYURL                       | The URL for the proxy for Amplify Central`<http://username:password@hostname:port>`. If empty, no proxy is defined.                                                                                                                                                                                                       |
 | central.usageReporting.url             | CENTRAL_USAGEREPORTING_URL             | The Lighthouse URL the agent publishes usage reports. See [Traceability usage reporting](#traceability-usage-reporting)                                                                                                                                                                                                   |
 | central.usageReporting.publish         | CENTRAL_USAGEREPORTING_PUBLISH         | Enables/disables the sending of usage events to Amplify (default: `true`)                                                                                                                                                                                                                                                 |
-| central.usageReporting.publishMetric   | CENTRAL_USAGEREPORTING_PUBLISHMETRIC   | Enables/disables the sending of metric events to Amplify (default: `true`)                                                                                                                                                                                                                                               |
+| central.usageReporting.publishMetric   | CENTRAL_USAGEREPORTING_PUBLISHMETRIC   | Enables/disables the sending of metric events to Amplify (default: `true`)                                                                                                                                                                                                                                                |
+| central.usageReporting.publishSubscriptionMetric   | CENTRAL_USAGEREPORTING_PUBLISHSUBSCRIPTIONMETRIC   | Enables/disables the sending of subscription metric events to Amplify (default: `false`)                                                                                                                                                                |
```

Comment:
Config is no longer needed as subscription/application metrics are combined with API metric, Left over by mistake, raised a PR https://github.com/Axway/agent-sdk/pull/374 to remove the config flag